packages/core/src/Table/stories/TableRenderers/TableRenderers.tsx

```diff
@@ -1985,6 +1990,23 @@ const DropdownColumnRenderer = () => {
 
 export const DropdownColumnRendererStory: StoryObj = {
   parameters: {
+    eyes: {
+      include: true,
+      runBefore() {
+        // reduce the number of visible rows
+        fireEvent.click(
+          screen.getByRole("combobox", { name: "Select how many to display" })
+        );
+        fireEvent.click(screen.getByRole("option", { name: "5" }));
+
+        const tableElement = screen.getByRole("table", {
+          name: "Severity",
+        });
+        fireEvent.click(within(tableElement).getByText("Major"));
```

Comment:
We're doing unnecessary work here (indirectly testing the Pagination); we could select the first "Major" with `getAllByText` without needing to "hide" the others

```suggestion
        const [firstMajor] = screen.getAllByText("Major");
        fireEvent.click(firstMajor);
```

Reply:
We need to do it because the issue only appears with few rows on the table. However the way to do it needs to be improved.
After talking with @MEsteves22 and with her help we decided we will start showing the table with only 5 elements to save the need for this interaction.

Reply:
The test was updated and the pagination starts with 5 ✅